internal/langserver/handlers/initialize_test.go

```diff
@@ -119,3 +120,42 @@ func TestInitialize_multipleFolders(t *testing.T) {
 	    ]
 	}`, rootDir.URI(), rootDir.URI())})
 }
+
+func TestInitialize_ignoreDirectoryNames(t *testing.T) {
+	tmpDir := TempDir(t, "plugin", "ignore")
+	pluginDir := filepath.Join(tmpDir.Dir(), "plugin")
+	emptyDir := filepath.Join(tmpDir.Dir(), "ignore")
+
+	InitPluginCache(t, pluginDir)
+	InitPluginCache(t, emptyDir)
+
+	ls := langserver.NewLangServerMock(t, NewMockSession(&MockSessionInput{
+		TerraformCalls: &exec.TerraformMockCalls{
+			PerWorkDir: map[string][]*mock.Call{
+				pluginDir: validTfMockCalls(),
+				emptyDir: {
+					// TODO! improve mock and remove entry for `emptyDir` here afterwards
```

Comment:
Do you mind creating an issue to track this and link it from this comment here?

Reply:
I've created issue #701 for this